### PR TITLE
Create FAQs.md

### PR DESCRIPTION
**Pull request by ckprice**
Getting this document started.

@johngruen @6a68 @javaun @meandavejustice feel free to add anything else.

### CODE REVIEW

**Comment by lmorchard** [comment]
Think I'll just merge this as soon as CircleCI is happy, so we have something to link to ASAP :)

File: docs/FAQs.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+### Why does Test Pilot require a Firefox Account?
+
+Test Pilot was launched with the requirement that particpants be logged into a Firefox Account.
+
+As this is an evolving product, this requirement may be adjusted in the future, but some of the original basis for this requirement is
+
+- We would like a way to inform participants in the future that new experiments are available.
+- Test Pilot is about building a community through FxA enabled forums such as Discourse.
+- We aslo use the account as a gate since we are doing certain metrics collections that we want to ensure people have knowingly elected to opt into.
```
Comment (lmorchard) [inline docs/FAQs.md]: Oh, but there is a typo, here... `s/aslo/also/`

File: README.md [hunk 2]
```diff
@@ -146,3 +146,5 @@ Make sure you clone the Test Pilot repo:
   * [`docs/WORK.md`](./docs/WORK.md) - information on how we create, triage and assign work.
 
   * [`docs/DEPLOYMENT.md`](./docs/DEPLOYMENT.md) - process for deploying Test Pilot to stage and production.
+
+  * [`docs/FAQs.md`](./docs/FAQs.md) - frequently asked questions.
```
Comment (lmorchard) [inline README.md:150]: 👍

**Comment by pdehaan** [comment]
Not to be "that guy" (but c'mon, we all know I'm that guy), but why not just use the [GitHub wiki](https://github.com/mozilla/testpilot/wiki)?
It wouldn't require Circle and PRs and is possibly more discoverable.

**Comment by ckprice** [comment]
> Not to be "that guy" (but c'mon, we all know I'm that guy), but why not just use the GitHub wiki?
> It wouldn't require Circle and PRs and is possibly more discoverable.

Let's file an issue on moving this to the Wiki (or at least discussing). For now, let's merge this just so we've got something.

**Comment by lmorchard** [comment]
Just saw the last comment after I already hit the button :/ But yeah, could be interesting to use the wiki, it's also interesting to solicit reviews & ship the doco in sync with the product

**Comment by chuckharmston** [comment]
> ship the doco in sync with the product

This is the big win for me. If we change something that requires a documentation change, those changes are made in step and on the record.

**Comment by meandavejustice** [comment]
Thank you @ckprice! This was much needed

**Comment by pdehaan** [comment]
Ref: https://github.com/mozilla/testpilot/issues/419 -- where we discuss removing the FxA restriction so people can view the site before having to sign up/in to FxA.

**Pull request merged**